lib/translator.js

```diff
@@ -36,19 +37,37 @@ Translator.prototype.getLocales = function() {
   return this._locales;
 };
 
+var createPoParseFn = function(lang) {
+  return function(localeDir) {
+    var fp = path.join(localeDir, lang, 'LC_MESSAGES', 'messages.po');
+    var po = '';
+    try{
+      po = fs.readFileSync(fp, {encoding: 'utf8'});
+    }catch(err) {
+      if(err.code !== 'ENOENT') {
+        throw err;
+      }
+    }
+    return gettextParser.po.parse(po).translations[''];
+  };
+};
+
+var createMsgParseFn = function(defaultLang, localeDirs) {
```

Comment:
A few critiques about the name `createMsgParseFn`.

`parseMessages` was a bad choice on my part.  All apologies.  We're not just parsing messages, we're doing more than that, we're parsing entire translation catalogs.  I should have used the name `parseCatalog`.

Avoid using the word `Fn` or `Function` when naming items, we'd never want to see `function parseFunction(){....`, we'd expect `function parser()` so lets name this guy `createCatalogParser` (or my personal pref is to use the prefix `get` vs `create`, it is shorter and I generally associate the word `create` with some sort of persistence layer).

Lets avoid anon functions and give these things names:

```js
function getCatalogParser(defaultLang, localeDirs) {
  var catalogParser = function(lang) {
    ...
  }
  return catalogParser;
}
```

This may be a bit more verbose but what we're doing here is super dense.

Thoughts?